src/data/python-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const pythonQuiz = [
+  {
+    Question: 
+      "In Python, which module is used to capture command-line arguments given at a file's runtime ?",
+    Answer: "sys",
+    Distractor1: "os",
+    Distractor2: "collections",
+    Distractor3: "itertools",
+    Explanation: 
+      "Python's sys module provides functions and variables used to manipulate different parts of the Python runtime environment, we can use sys.argv to get a list of command line arguments.",
```

Comment:
Let's remove the first part of the sentence since it is really similar to the Geeks for Geeks definition. 
```suggestion
      "sys.argv is used to get a list of command line arguments.",
```